galaxy/values-cvmfs.yaml

```
@@ -213,63 +213,72 @@ useSecretConfigs: false
+          {{- if .Values.extraVolumes }}
+            {{- range $num, $entry := .Values.extraVolumes }}
+              {{- if $entry.name }}
+                {{- if $entry.persistentVolumeClaim}}
+                  {{- if $entry.persistentVolumeClaim.claimName }}
+                    {{- range $num, $mount := $.Values.extraVolumeMounts }}
+                      {{- if $mount.name }}
+                        {{- if (eq $entry.name $mount.name) }}
+                          {{- if $mount.mountPath -}}
+                            ,{{- $entry.persistentVolumeClaim.claimName -}}:{{- $mount.mountPath -}}
                           {{- end }}
                         {{- end }}
                       {{- end }}
                     {{- end }}
                   {{- end }}
                 {{- end }}
-              {{- end -}}
-              </param>
-              <param id="k8s_namespace">{{ .Release.Namespace }}</param>
-              <!-- Must be DNS friendly and less than 20 characters -->
-              <param id="k8s_galaxy_instance_id">{{ .Release.Name }}</param>
-              <param id="k8s_run_as_user_id">101</param>
-              <param id="k8s_run_as_group_id">101</param>
-              <param id="k8s_fs_group_id">101</param>
-              <param id="k8s_supplemental_group_id">101</param>
-              <param id="k8s_pull_policy">IfNotPresent</param>
-              <param id="k8s_cleanup_job">onsuccess</param>
-              <param id="k8s_pod_priority_class">{{ include "galaxy.fullname" . }}-job-priority</param>
-            </plugin>
-        </plugins>
-        <handlers assign_with="db-skip-locked" />
-        <destinations default="dynamic-k8s-dispatcher">
-            <destination id="local" runner="local"/>
-            <destination id="dynamic-k8s-dispatcher" runner="dynamic">
-              <param id="type">python</param>
-              <param id="function">k8s_container_mapper</param>
-              <param id="docker_default_container_id">{{ .Values.image.repository }}:{{ .Values.image.tag }}</param>
-              <param id="docker_enabled">true</param>
-            </destination>
-        </destinations>
-        <limits>
-            <limit type="registered_user_concurrent_jobs">5</limit>
-            <limit type="anonymous_user_concurrent_jobs">2</limit>
-        </limits>
-    </job_conf>
+              {{- end }}
+            {{- end }}
+          {{- end }}
+        k8s_namespace: "{{ .Release.Namespace }}"
+        k8s_pod_retries: "4"
+        k8s_walltime_limit: "604800"
+        k8s_galaxy_instance_id: "{{ .Release.Name }}"
+        #k8s_timeout_seconds_job_deletion: 30
+        k8s_supplemental_group_id: "101"
+        k8s_fs_group_id: "101"
+        k8s_pull_policy: IfNotPresent
+        k8s_cleanup_job: onsuccess
+        k8s_pod_priority_class: '{{ include "galaxy.fullname" . }}-job-priority'
+        k8s_run_as_user_id: "101"
+        k8s_run_as_group_id: "101"
```

Just a minor cosmetic improvement to group user id related stuff together - moves run_as_user_id and run_as_group_id to the top.
```suggestion
        k8s_run_as_user_id: "101"
        k8s_run_as_group_id: "101"
        k8s_supplemental_group_id: "101"
        k8s_fs_group_id: "101"
        k8s_pull_policy: IfNotPresent
        k8s_cleanup_job: onsuccess
        k8s_pod_priority_class: '{{ include "galaxy.fullname" . }}-job-priority'
```